hadoop-ozone/client/src/test/java/org/apache/hadoop/ozone/client/MultiNodePipelineBlockAllocator.java

```diff
@@ -85,4 +110,34 @@ public MultiNodePipelineBlockAllocator(OzoneConfiguration conf,
         .setLength(blockSize).build());
     return results;
   }
+
+  private void addMembers(HddsProtos.Pipeline.Builder builder,
+      int nodesNeeded, Set<DatanodeDetails> datanodes,
```

Comment:
nit: it would better explain the use if the datanodes parameter would be called as excludedDataNodes, or excludedNodes.

Reply:
Yeah, it should be excludedDataNodes for better readbility. I have just updated, thanks for pointing.